source/atlas-data-lake-testing/tests/crud/getMore.yml

```diff
@@ -0,0 +1,32 @@
+collection_name: &collection_name "driverdata"
+database_name: &database_name "test"
+
+tests:
+  -
+    description: "A successful find event with getMore"
+    operations:
+      -
+        object: collection
+        name: find
+        arguments:
+          filter: { a: { $gte: 2 }}
+          sort: { a: 1 }
+          batchSize: {"$numberLong": "3"}
+          limit: {"$numberLong": "4"}
+    expectations:
+      -
+        command_started_event:
+          command:
+            find: *collection_name
+            filter: { a: { $gte : 2 }}
+            sort: { a: 1 }
+            limit: 4
+            batchSize: 3
+          command_name: "find"
+          database_name: *database_name
+      -
+        command_started_event:
+          command:
+            batchSize: 1
+          command_name: "getMore"
+          database_name: "cursors"
```

Comment:
Looks like this typo persisted from my previous review: https://github.com/mongodb/specifications/pull/821#discussion_r440966275

Reply:
Yes... that is currently what we send back. It might not always be what we send back, so I'm hesitant to suggest that you validate it. But for now, (and maybe forever), it is "cursors".

The collection name is autogenerated, so there isn't anyway to validate that.